admin/public/styles/keystone.less

```diff
@@ -113,9 +113,10 @@
 
 .DayPicker-Weekdays {
 	display: table-header-group;
-	// > div {
-	//	 display: table-row;
-	// }
+}
+
+.DayPicker-WeekdaysRow {
+	display: tahle-row;
```

Comment:
Should this be `display: table-row;`

Reply:
Yes it should have been. Thank you for catching that.